code-challenges.js

```diff
@@ -1,62 +1,126 @@
-// Expected output: "350 is above boiling point"
-
-const temperature3 = 212
-// Expected output: "212 is at boiling point"
-
+// const temperature1 = 42
+// // Expected output: "42 is below boiling point"
+// const temperature2 = 350
+// // Expected output: "350 is above boiling point"
+// const temperature3 = 212
+// // Expected output: "212 is at boiling point"
 // Pseudo code:
-
-// --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
-
-const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
-const padres1998WorldSeriesRuns = [6, 3, 5, 3]
-// Expected output: 9
-
+// name of function: hotAndCold
+// input: numbers that represent tempuratures
+// output: adding the description of whether the tempuratures are either @boiling, above boiling, or below boiling to each numerical tempurture
+// process: "if" conditional statement, assigning descriptions to specific variables
+// const temps = [42, 350, 212]
+// if (temps === 212) {
+//     console.log("212 is at boiling point.")
+// }
+const hotAndCold = (tempurature) => {
+    if (tempurature >= 212){
+        return "is at boiling point."
+}
+else if (tempurature >= 350){
+    return "is above boiling point."
+}
+else if (tempurature <= 42){
+    return "is below boiling point."
+}
+else {
+    return "error"
+    }
+}
+console.log(hotAndCold(42))
+console.log(hotAndCold(212))
+console.log(hotAndCold(350))
+// NOTES: The terminal is stating "ReferenceError: cannot access 'array' before initialization" - I'm not sure what I've done wrong - to generate the outputs I first tried "tempurature" instead of "hotAndCold" within the console.log but that also did not work.
+// // --------------------2) Create the code that will combine the two arrays and return the length using the test variables provided below.
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// // Expected output: 9
 // Pseudo code:
-
-// --------------------3) Create the code that will reverse the letters of a string using the test variable provided below.
-
-const currentCohort = "Foxtrot 2023"
+// name of function: competitions
+// input: 2 arrays of numbers
+// output: the amount of variables within both arrays combined
+// process: combine values within arrays and return the amount of characters of the combined values
+// const padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+// const padres1998WorldSeriesRuns = [6, 3, 5, 3]
+// const competitions = padres1984WorldSeriesRuns.concat.(padres1998WorldSeriesRuns){
+// }
+// console.log(competitions.length)
+// competitions.length
+var padres1984WorldSeriesRuns = [2, 5, 2, 2, 4]
+var padres1998WorldSeriesRuns = [6, 3, 5, 3]
+console.log(padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns))
+    let array = [array.length]
+//NOTES: I keep getting an error message, I think I'm on the right track but there's probably a lot wrong with my syntax?
```

Comment:
You are definitely on the right tracks. Allow your variable to store the concat() method you used to combine the arrays then perform .length on the variable.
```js
let array = padres1984WorldSeriesRuns.concat(padres1998WorldSeriesRuns)
console.log(array.length)
```